src/Sulu/Bundle/TestBundle/Testing/SuluTestCase.php

```diff
@@ -26,7 +26,7 @@
 use Symfony\Component\Security\Core\Tests\Authentication\Token\TestUser;
 
 /**
- * Base test case for functional tests in Sulu.
+ * @deprecated use Sulu\Bundle\TestBundle\TestCase\SuluTestCase instead
```

Comment:
Shouldn't we get rid of that class now, if we want to move it? Resp. why do we keep the `KernelTestCase` in this directory, but move the other one?

Reply:
this was too early didn't want to deprecate this class now